Assets/Scripts/ScoreHandler/ScoreHandler.cs

```diff
@@ -15,12 +15,12 @@ public int CalculateScore(string word)
 
         if (word != string.Empty)
         {
-            var dictionary = LetterDictionary.getLetters();
+            var letters = LetterList.GetLettersAsDictionary();
```

Comment:
Scuffed but allow it

Reply:
How would you suggest getting the score ? convert the word into "Letters" - the custom class?

Reply:
Refactor a bunch of stuff, which is alot more work then its worth right now

Reply:
Probably should have started with a Letter object from the beginning and pass objects around instead of characters, but its too late for that - this is a decent solution for what we need